src/Command/Pull/PullCommandBase.php

```diff
@@ -733,6 +732,7 @@ protected function runDrushSqlSanitize(\Closure $output_callback): void {
         throw new AcquiaCliException('Unable to sanitize Drupal database via Drush. {message}', ['message' => $process->getErrorOutput()]);
       }
       $this->checklist->completePreviousItem();
+      $this->io->note('Your database was sanitized via sql:sanitize. This has changed all user passwords to randomly generated strings.');
```

Comment:
@anavarre

Reply:
I'd just do "Your database was sanitized for security reasons. This has changed all user passwords to randomly generated strings. To log in to your Drupal site, use drush uli"

Reply:
I'd like to refer to sql:sanitize so they know how it was done, especially since it does more that just sanitize passwords and it's extendable.

Reply:
Ok, can we still explain how to log in to the site?

Reply:
Sure.